start-as-experiment.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Start as an Experiment
+
+# Problem
+
+An InnerSource initiative is considered but not started because management is 
+unsure about its outcome and, as a result, is not willing to commit to an 
+investment.
+
+# Context
+
+The majority of software development projects in the company are distributed
+globally. The company is considering to leverage InnerSource to make the
+required collaboration more efficient. However, most managers are not familiar
+with the Open Source working model and are instead accustomed to hierarchical,
+top-down control style management. The idea of InnerSource is very popular with
+software developers in the company, not the least because many developers use
+or are actively developing Open Source software.
+
+# Forces
+
+- Managers will want to validate the claims of improved collaboration through
+  InnerSource before making a long term investment. This usually involves
+  putting numbers against the expected improvements, which are hard to come
+  by for InnerSource.
+- If the InnerSource initiative has a huge uptake among developers and if many
+  projects come to rely on it, a decision to shut it down will be very 
+  unpopular and therefore hard to make.
+- When implementing new working models such as InnerSource which are radically
+  different from working models previously practiced in the company, it is 
+  likely that existing, mandatory processes are no longer applicable and even
+  that one has to operate in a regulatory, sometimes legal no-mans land.
```

Comment:
I know of three areas, which were (are) problematic, at least here in Europe:
- Tax problems (a project in one country contributing SW for free, a project in another country makes money of of it)
- Open Source licensing obligations: In my company, there are strict processes ensuring that SW that we publish and that uses OSS complies with the requirements spelled out in their respective licenses. Technically, pushing SW to a platform that is accessible by multiple legal entities is already an act of publishing. I'm guessing that most companies which haven't practiced InnerSource do not have regulations for that in place.
- Export control regulations: There are a lot of export control regulations (most of them put in place by the US) that apply to SW being published (same case as above).